src/key.js

```diff
@@ -92,11 +93,11 @@ class Key {
 
     let numericKey = new Key({ note: 1 });
     let symbolKey = key.clone();
-    const reference = this.clone().normalize().useModifier(key.modifier);
+    const reference = this.clone().normalize().useModifier(key.modifier).normalizeEnharmonics(key);
```

Comment:
Like here for example. If useModifier() was to be the last called function, it wouldn't be normalized right. So if we combine normaize() and normalizeEnharmonics(), we gotta make sure useModifier() isn't the last thing called.